daml/Tests/Trading/TradingIntegrationTest.daml

```diff
@@ -1,14 +1,14 @@
 module Tests.Trading.TradingIntegrationTest where
```

Comment:
Mmm we should remove this file if it's going to be completely commented out. @brian-weir I recall you used this for your integration tests and it will only pass when you have the adapter up, is that right? On the other hand, I do understand that we want `daml test` to pass in the CI.

Reply:
@brian-weir 's suggestion was to move it out of the source tree that's being compiled. I think that would be a sensible option if we can get it to work as part of `daml test`. I can revert the commenting out for now and leave it for Brian to fix.

Reply:
I've reverted this for now